### PR TITLE
Fix for bug #216

### PR DESCRIPTION
**Pull request by Wakie87**
#### What's this PR do?
This PR will allow user to enter drugs that have combinations
#### Where should the reviewer start?
Adding prescription/drug for a patient.
#### How should this be manually tested?
Add a prescription/drug for a patient. In the size box enter a numbers (eg.  5/325, 1000/12.5/32) with dots and backslash. preg_replace is used so everything else is stripped.
#### Any background context you want to provide?
combination medicines which are noted in two strengths, one for the DrugA and one for DrugB (e.g. FancyDrugName 10/325, FancyDrugName 5/325, FancyDrugName 7.5/325, etc)
#### What are the relevant tickets?
Issue #216 
Original PR #28 
#### Questions:

### CODE REVIEW

**Comment by fr4nkie** [comment]
Thanks Wakie. I appreciate you cleaning this up.

File: sql/4_2_2-to-5_0_0_upgrade.sql [hunk 1]
```diff
@@ -771,3 +771,13 @@ INSERT INTO `supported_external_dataloads` (`load_type`, `load_source`, `load_re
 ALTER TABLE `form_encounter` ADD `pos_code` tinyint(4) default NULL;
 #EndIf
 
+#IfNotColumnType drugs size varchar
+ALTER TABLE `drugs` CHANGE `size` `size` varchar(25) default NULL ;
+#EndIf
+
+#IfColumn prescriptions size
```
Comment (bradymiller) [inline sql/4_2_2-to-5_0_0_upgrade.sql]: Can you use IfNotColumnType here. maybe:
``````
#IfColumnType prescriptions size varchar(25)

Comment (bradymiller) [inline sql/4_2_2-to-5_0_0_upgrade.sql]: Note I am not sure if above will work.

File: library/classes/Prescription.class.php [hunk 2]
```diff
@@ -300,16 +300,15 @@ function get_refills() {
     }
 
     function set_size($size) {
-        if (is_numeric($size)) { $this->size = $size; }
+        $this->size = preg_replace("/[^0-9\/\.]/", "",$size);
```
Comment (bradymiller) [inline library/classes/Prescription.class.php]: lets add a hyphen in here also; sometimes see them separate by a:
``````
-

**Comment by Wakie87** [comment]
Commited to the codebase. Thanks for your code contribution @fr4nkie

**Pull request closed without merging**